apps/dashboard/app/javascript/xdmod.js

```diff
@@ -2,12 +2,13 @@
 import _ from 'lodash';
 import {xdmodUrl, analyticsPath} from './config';
 import {today, startOfYear, thirtyDaysAgo} from './utils';
-import { jobsPanel } from './xdmod/jobs';
+import { jobsPanel, jobAnalyticsHtml } from './xdmod/jobs';
```

Comment:
I'm not sure I'm in love with the way this is written almost as a separate thing that's additive to `jobsPanel`. Seems to me like `jobsPanel` should have encapsulated all this work. 

Can't it? It creates these divs, can't it attach event listeners to them at that time?

I just feel like a lot of these additions should have actually been held within `xdmod/jobs.js` rather than exporting an entirely new function that adds to/modifies the result of the existing one.

Reply:
Fixed